### PR TITLE
Refine Position

### PR DESCRIPTION
**Pull request by Jeff-CCH**
- [x] refine Position
- [x] refine collateral swap e2e test

### CODE REVIEW

File: src/handlers/v1/markets/[chainId]/[marketId]/deleverage/index.ts [hunk 1]
```diff
@@ -143,8 +141,10 @@ export const v1GetDeleverageQuotationRoute: Route<GetDeleverageQuotationRoutePar
       targetPosition = {
         utilization: calcUtilization(targetBorrowCapacityUSD, targetBorrowUSD),
         healthRate: calcHealthRate(targetCollateralUSD, targetBorrowUSD, targetLiquidationThreshold),
+        liquidationThreshold: targetLiquidationThreshold,
+        borrowUSD: common.formatBigUnit(targetBorrowUSD, 2),
+        collateralUSD: targetCollateralUSD.toString(),
```
Comment (zodahu) [inline src/handlers/v1/markets/[chainId]/[marketId]/deleverage/index.ts:146]:
```suggestion
        collateralUSD: common.formatBigUnit(targetCollateralUSD, 2),
```

File: src/handlers/v1/markets/[chainId]/[marketId]/collateral-swap/index.ts [hunk 2]
```diff
@@ -135,8 +134,10 @@ export const v1GetCollateralSwapQuotationRoute: Route<GetCollateralSwapQuotation
       targetPosition = {
         utilization: calcUtilization(targetBorrowCapacityUSD, targetBorrowUSD),
         healthRate: calcHealthRate(targetCollateralUSD, targetBorrowUSD, targetLiquidationThreshold),
+        liquidationThreshold: targetLiquidationThreshold,
+        borrowUSD: common.formatBigUnit(targetBorrowUSD, 2),
+        collateralUSD: targetCollateralUSD.toString(),
```
[the remark this answers is not in the document]
Comment (Jeff-CCH) [reply src/handlers/v1/markets/[chainId]/[marketId]/collateral-swap/index.ts:139]: Should we keep `borrowUSD: common.formatBigUnit(targetBorrowUSD, 2)` to be consistent with other index.ts?

[the remark this answers is not in the document]
Comment (zodahu) [reply src/handlers/v1/markets/[chainId]/[marketId]/collateral-swap/index.ts:139]: It's fine with me.

**Review by hihiben — APPROVED** [review]
LGTM

**Pull request merged**